### DISCUSSION TITLE
Discussions category/tag for Kudos/Thank you.

### DISCUSSION THREAD

**Discussion by MarkJaroski** [keepassxreboot/keepassxc#11422] [Ideas]
I just upgraded to the newest release and the tiny quality-of-life improvements add up to make it a remarkable overall improvement. Thank everyone for all of your work.

I'm posting this under ideas, because there is no discussion category for praise for our developers and contributors. Maybe there should be one?

**Comment by droidmonkey** [comment]
There was one that may have been pushed down. Thank you, though!!
